websites/docs-smoke-test/src/releases/release-format-definition.mdx

```diff
@@ -5,11 +5,25 @@ description: |
   The description must be a stand-alone text conveying the key change and its advantages for external feed consumers,
   redistribution, notifications and others.
 type: feature # feature | fix | enhancement | announcement
-topics:
-  - Bar
-  - FooBar
-#  - Baz
-#  - Foo
+topics: # see topic examples below
```

Comment:
I've added a couple of topics to the release note so that we can display enough topics in the filter to trigger the scrolling